### ISSUE TITLE
'The planemo image is not ready'

### ISSUE THREAD

**Issue by Ba28** [kellrott/galaxy-tooldev-docs#19]
Once I received access to the image, another error came up:

ERROR: (gcloud.compute.instances.create) Some requests did not succeed:
- The resource 'projects/enhanced-layout-94320/global/images/planemo-machine-image' is not ready

Not sure what this means or how to interpret it.

**Comment by kellrott** [comment]
We haven't seen this error before. Did it happen if you repeated launching the step? Maybe there is a lag time between loading the image and it actually being ready to launch a VM

**Comment by Ba28** [comment]
Yes, it happened when I was repeating the steps.  It's still giving me the error.  When I retried the previous commands before this, it just says the resource already exists (even though I never saw a successful output message).

**Comment by kellrott** [comment]
On the Console, can you go to Compute -> Images and then delete all of the Planemo images? 
Maybe you have a half loaded images that is causing problems.
